timesketch/frontend-ng/src/components/Visualization/ChartCard.vue

```diff
@@ -0,0 +1,186 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+ <template>
+  <v-card
+    flat
+    :min-height="height"
+    :width="width + (chartType === 'table' ? 25 : 0)"
+  >
+    <v-card-text>
+      <TsBarChart 
+        v-if="chartSeries != null && chartType === 'bar'"
+        :fieldName="fieldName"
+        :metricName="metricName"
+        :is-time-series="isTimeSeries"
+        :chartSeries="chartSeries" 
+        :chartLabels="chartLabels"
+        :chartTitle="chartTitle"
+        :height="height"
+        :width="width"
+        :xTitle="xTitle"
+        :showXLabels="showXLabels"
+        :yTitle="yTitle"
+        :showYLabels="showYLabels"
+        :showDataLabels="showDataLabels"
+      >
+      </TsBarChart>
+      <TsColumnChart
+        v-if="chartSeries != null && chartType === 'column'"
+        :fieldName="fieldName"
+        :metricName="metricName"
+        :is-time-series="isTimeSeries"
+        :chartSeries="chartSeries" 
+        :chartLabels="chartLabels"
+        :chartTitle="chartTitle"
+        :height="height"
+        :width="width"
+        :xTitle="xTitle"
+        :showXLabels="showXLabels"
+        :yTitle="yTitle"
+        :showYLabels="showYLabels"
+        :showDataLabels="showDataLabels"
+      >
+      </TsColumnChart>
+      <TsDonutChart
+        v-if="chartSeries != null && chartType === 'donut'"
+        :fieldName="fieldName"
+        :metricName="metricName"
+        :is-time-series="isTimeSeries"
+        :chartSeries="chartSeries" 
+        :chartLabels="chartLabels"
+        :chartTitle="chartTitle"
+        :height="height"
+        :width="width"
+        :xTitle="xTitle"
+        :showXLabels="showXLabels"
+        :yTitle="yTitle"
+        :showYLabels="showYLabels"
+        :showDataLabels="showDataLabels"
+      >
+      </TsDonutChart>
+      <TsHeatmapChart
+        v-if="chartSeries != null && chartType === 'heatmap'"
+        :fieldName="fieldName"
+        :metricName="metricName"
+        :is-time-series="isTimeSeries"
+        :chartSeries="chartSeries" 
+        :chartLabels="chartLabels"
+        :chartTitle="chartTitle"
+        :height="height"
+        :width="width"
+        :xTitle="xTitle"
+        :showXLabels="showXLabels"
+        :yTitle="yTitle"
+        :showYLabels="showYLabels"
+        :showDataLabels="showDataLabels"
+      >
+      </TsHeatmapChart>
+      <TsLineChart 
+        v-if="chartSeries != null && chartType === 'line'"
+        :fieldName="fieldName"
+        :metricName="metricName"
+        :is-time-series="isTimeSeries"
+        :chartSeries="chartSeries" 
+        :chartLabels="chartLabels"
+        :chartTitle="chartTitle"
+        :height="height"
+        :width="width"
+        :xTitle="xTitle"
+        :showXLabels="showXLabels"
+        :yTitle="yTitle"
+        :showYLabels="showYLabels"
+        :showDataLabels="showDataLabels"
+      >
+      </TsLineChart>
+      <TsTableChart
```

Comment:
This might be something we do not need to "fix" or there might be no fix for it. But I noticed that when asking for the 10 most rare domains, I get 29 results (17 with count 1).

![image](https://github.com/google/timesketch/assets/99879757/d6afba64-ee59-44ef-ba47-959ec1495c26)

Reply:
yeah this is WAI.  max document count for rare terms works differently to top terms in that it returns results for terms that have document counts up to the specified max value (10 in your example)

Reply:
Oh, okay. I need to check if the field explanation makes this clear for the user.

Reply:
You are right it should show only up to 3 documents per domain.  Marking this is fixed as it appears it was related to the `maxItems` prop issue.